server/game_service.py

```diff
@@ -210,5 +216,22 @@ def __getitem__(self, item: int) -> Game:
     def __contains__(self, item):
         return item in self._games
 
-    async def send_to_rating_service(self, summary: GameRatingSummary):
-        await self._rating_service.enqueue(summary)
+    async def publish_game_results(self, game_results: EndedGameInfo):
+        result_dict = game_results.to_dict()
+        await self._message_queue_service.publish(
+            "game_results",
+            ".".join([
+                "results",
+                result_dict["rating_type"],
+                result_dict["featured_mod"],
+                result_dict["validity"],
+            ]),
+            result_dict,
+        )
+
```

Comment:
Selecting a useful topic is not easy :smile: What is contained in rating_type?

In general at work we came up with the following approach:

## "The shitting camels" 
Use dots to distinguish sub-parts, use camel case between the dots.
(The dots allow wildcard filtering on certain parts of the routing key)

Example: request.meterData.create

**Routing key names:**
Schema: \<messageType>.\<entity>.\<operation>

**Message type** can be:
- request → messages intending to perform a state change
- success → confirmation/notification of a state change
- error → notification about a denied state change
  an error is like a 4xx HTTP status code: inform the sender about an error
  an error is not like 5xx HTTP status code: in case of "internal" errors, use the dead letter queue

**Entity** describes the object that is being operated on

**Operation**:
  - create
  - update
  - delete
  - use update if no distinction is possible (e.g. on ingestion)
  - \<to be discussed>

**Examples:**
- request.meter.create → success.meter.create
- request.meter.update → error.meter.update